src/scss/foundations/colors.scss

```diff
@@ -0,0 +1,119 @@
+@use "sass:color";
+@use "../lib";
+
+$_colorActionBase: #067f96;
+$_colorActionDark: #e5f1f4;
+$_colorActionLight: #045767;
+
+$_colorAlertBase: #ffdc50;
+$_colorAlertDark: #fff8dc;
+$_colorAlertLight: #ffc010;
+
+$_colorBrandBase: #8334c2;
+$_colorBrandDark: #f2eaf8;
+$_colorBrandLight: #5b2488;
+
+$_colorErrorBase: #df1654;
+$_colorErrorDark: #fbe7ed;
+$_colorErrorLight: #b21143;
+
+$_colorNeutral25: #595d6a;
+$_colorNeutral50: #90939d;
+$_colorNeutral75: #c7c9cd;
+$_colorNeutral85: #dddee1;
+$_colorNeutral95: #f3f4f4;
+$_colorNeutralBase: #21283b;
+$_colorNeutralLight: #ffffff;
+
+$_colorSuccessBase: #1d8147;
+$_colorSuccessDark: #e8f2ec;
+$_colorSuccessLight: #145a31;
+
+/*
+The color palette of Pyxis
+
+@type Map
+
+@access private
+*/
+
+$_colors: (
+  action: (
+    base: $_colorActionBase,
+    dark: $_colorActionDark,
+    light: $_colorActionLight,
+  ),
+  alert: (
+    base: $_colorAlertBase,
+    dark: $_colorAlertDark,
+    light: $_colorAlertLight,
+  ),
+  brand: (
+    base: $_colorBrandBase,
+    dark: $_colorBrandDark,
+    light: $_colorBrandLight,
+  ),
+  error: (
+    base: $_colorErrorBase,
+    dark: $_colorErrorDark,
+    light: $_colorErrorLight,
+  ),
+  neutral: (
+    25: $_colorNeutral25,
+    50: $_colorNeutral50,
+    75: $_colorNeutral75,
+    85: $_colorNeutral85,
+    95: $_colorNeutral95,
+    base: $_colorNeutralBase,
+    light: $_colorNeutralLight,
+  ),
+  success: (
+    base: $_colorSuccessBase,
+    dark: $_colorSuccessDark,
+    light: $_colorSuccessLight,
+  ),
+);
+
+/*
+Returns the $color hex with a $variant from map $_colors
+
+@param {String} $color - Name of color
+@param {String} $variant - Name of variant associated with $color, by default is "base"
+
+@return {Color} Representation of named color
+@throws {Error}
+
+@access public
+*/
+
+@function color($color, $variant: base) {
+  @if not map-has-key($_colors, $color) {
+    @error lib.keyNotFoundError("$_colors", $color);
+  }
+
+  @if not map-has-key($_colors, $color, $variant){
+    @error lib.keyNotFoundError("$_colors", $color, $variant);
+  }
+
+  @return map-get($_colors, $color, $variant);
+}
```

Comment:
Perché invece che fare una funzione in cui è possibile specificare qualsiasi colore non usiamo un mixin che torni di default o `color: COLORE` o `background-color: COLORE` cosi da impedire che si possano creare combinazioni o usare colori non previsti dal DS? Anche due funzioni separate se non ha senso averne una sola